src/server/Cypher.js

```diff
@@ -69,7 +69,14 @@ router.post("/", async (req, res) => {
       isSchemaChanged =
           JSON.stringify(schema) !== JSON.stringify(currentSchema);
     }
-    return res.send({ rows, dataTypes, isSchemaChanged });
+    const replacer = (key, value) => {
```

Comment:
Let's add a comment before this function to note that this is to solve the JSON stringify issue for INT128 type.